utils/appliance.py

```diff
@@ -1173,6 +1174,7 @@ def restart_evm_service(self, rude=False, log_callback=None):
                 msg = 'Failed to restart evmserverd on {}\nError: {}'.format(self.address, msg)
                 log_callback(msg)
                 raise ApplianceException(msg)
+                fire("server_details_changed")
```

Comment:
This doesn't seem to be useful here. ```raise``` is going to immediately return flow control back to the calling function, and hence line 1177 will never be called. Even if the Exception is caught in the calling function, this will not work. Perhaps you are expecting this to be tabbed out 2 indents?